app/routes.py

```diff
@@ -1,2 +1,53 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify
+
+class Planet:
+    def __init__(self, id, name, description, color):
+        self.id = id
+        self.name = name 
+        self.description = description
+        self.color = color
+    
+    def return_planets(self):
+        return {
+            "id": self.id,
+            "name": self.name,
+            "description": self.description,
+            "color": self.color  
+        }
+        
+
+planets = [
+    Planet(1,"Mercury", "hot", "brown"), 
+    Planet(2, "Venus", "pretty", "redish brown"),
+    Planet(3, "Earth", "wet", "greenis blue"),
+    Planet(6, "Saturn", "rings", "purple and yellow" )
+
+]
+    
+
+solar_systems_bp = Blueprint("planets", __name__, url_prefix="/planets")
```

Comment:
Minor note: for readability, consider removing extra line spaces.
```suggestion
        }
        
planets = [
    Planet(1,"Mercury", "hot", "brown"), 
    Planet(2, "Venus", "pretty", "redish brown"),
    Planet(3, "Earth", "wet", "greenis blue"),
    Planet(6, "Saturn", "rings", "purple and yellow" )
]
    
solar_systems_bp = Blueprint("planets", __name__, url_prefix="/planets")
```